src/code/z_skelanime.c

```diff
@@ -1716,7 +1729,7 @@ s32 SkelAnime_LoopPartial(SkelAnime* skelAnime) {
     }
 
     SkelAnime_AnimateFrame(skelAnime);
-    return 0;
+    return false;
```

Comment:
This function is used as a callback for `skelAnime->update.normal`. In `SetUpdate` you can see other callbacks set for this member
```c

/**
 * Sets the SkelAnime's update function based on its current mode.
 */
void SkelAnime_SetUpdate(SkelAnime* skelAnime) {
    if (skelAnime->mode <= ANIMMODE_LOOP_INTERP) {
        skelAnime->update.normal = SkelAnime_LoopFull;
    } else if (skelAnime->mode <= ANIMMODE_ONCE_INTERP) {
        skelAnime->update.normal = SkelAnime_Once;
    } else {
        skelAnime->update.normal = SkelAnime_LoopPartial;
    }
}
```

The update function of a skelanime returns `true` when it has finished playing and `false` if it is still playing, and since this function is a loop it will not stop, so will never return `true`

Reply:
Ah okay thanks! Being part of a callback makes sense